magicctapipe/conftest.py

```diff
@@ -1030,7 +1031,7 @@ def real_dl3_monly(real_dl2_monly, IRF_monly, temp_DL3_monly, config_monly):
 
 
 @pytest.fixture(scope="session")
-def real_index(real_dl3):
+def real_index(real_dl3, temp_DL3):
```

Comment:
the usage of temp_DL3 in here looks confusing. 
temp_DL3 executes a fixture that creates the output directory, but you create the index file not in this temp_DL3 directory, but in real_dl3, yet you return the path of the temporary directory. 
so later on in test_exist_index(real_index) you do not check the directory of "real_dl3", but the one of of temp_DL3. Either there is something wrong with this test, or it needs more explanations what you are really testing

Reply:
fixed